spec/v2/index.bs

```diff
@@ -493,14 +493,21 @@ Advisement:
         <td>String
         <td>M
         <td>
-            The IPCC version of the GWP characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be one of the following:
+          **Deprecated**
+
+          The IPCC version of the GWP characterization factors used in the calculation of the largest relative share of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be one of the following:
```

Comment:
@raimundo-henriques Why the addition of "of the largest relative share"? Especially given teh attribute would be deprecated, why this addition?